samples/java/src/main/resources/application.conf

```diff
@@ -0,0 +1,19 @@
+# #akka_config
+akka {
+  actor {
+    provider = "remote"
+  }
+
+  remote {
+    artery {
+      enabled = on
+      transport = tcp
+
+      # This is the hostname this instance uses to advertise itself to peers
+      canonical.hostname = "localhost"
+      canonical.port = 0
+    }
+    log-remote-lifecycle-events = off
+  }
+}
+# #akka_config
```

Comment:
can you add a newline at the end of file so that the github warning goes away

Reply:
added new line